data/news/napa.json

```diff
@@ -10,6 +10,433 @@
       "date_published": "2020-10-15T00:00:00-07:00",
       "summary": "Every county in California is assigned to a colored tier of the State\u2019s COVID-19 Blueprint for a Safer Economy, based on its test positivity, adjusted case rate and health equity metric. Napa County is currently in the red tier but ma..."
     },
+    {
+      "id": "https://www.countyofnapa.org/CivicAlerts.aspx?AID=240",
+      "url": "https://www.countyofnapa.org/CivicAlerts.aspx?AID=240",
+      "title": "Napa County on Track to Move to Orange Tier",
+      "date_published": "2020-10-15T00:00:00-07:00",
+      "summary": "Every county in California is assigned to a colored tier of the State\u2019s COVID-19 Blueprint for a Safer Economy, based on its test positivity, adjusted case rate and health equity metric. Napa County is currently in the red tier but ma..."
+    },
+    {
+      "id": "https://www.countyofnapa.org/CivicAlerts.aspx?AID=240",
+      "url": "https://www.countyofnapa.org/CivicAlerts.aspx?AID=240",
+      "title": "Napa County on Track to Move to Orange Tier",
+      "date_published": "2020-10-15T00:00:00-07:00",
+      "summary": "Every county in California is assigned to a colored tier of the State\u2019s COVID-19 Blueprint for a Safer Economy, based on its test positivity, adjusted case rate and health equity metric. Napa County is currently in the red tier but ma..."
+    },
```

Comment:
This repetition is actually happening on Napa County’s website (every news story is getting repeated 33 times), so this isn’t exactly our fault, but seems like a kind of madness we need to fix.